projects/angular/src/accordion/accordion-panel.spec.ts

```diff
@@ -36,6 +39,40 @@ class TestComponent {
   open = false;
   disabled = false;
   showDescription = false;
+  showHeading = false;
+  headingLevel: HeadingLevel = 3;
+  change = state => {
+    return state;
+  };
+}
+
+@Component({
+  template: `
+    <clr-accordion>
+      <clr-accordion-panel
+        [clrAccordionPanelOpen]="true"
+        [clrAccordionPanelHeadingEnabled]="true"
+        (clrAccordionPanelOpenChange)="change($event)"
+      >
+        <clr-accordion-title>title 1</clr-accordion-title>
+        <clr-accordion-content>
+          <clr-accordion>
+            <clr-accordion-panel
+              id="nested-accordion-panel"
+              [clrAccordionPanelOpen]="true"
+              [clrAccordionPanelHeadingEnabled]="true"
+              (clrAccordionPanelOpenChange)="change($event)"
+            >
+              <clr-accordion-title>nested title</clr-accordion-title>
+              <clr-accordion-content>nested panel</clr-accordion-content>
+            </clr-accordion-panel>
+          </clr-accordion>
+        </clr-accordion-content>
+      </clr-accordion-panel>
+    </clr-accordion>
+  `,
+})
+class TestNestedAccordionComponent {
   change = state => {
     return state;
   };
```

Comment:
This should be removed in the `TestNestedAccordionComponent` because it is not used.

Reply:
Corrected in commit: d9b21e4